spec/models/spell_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Spell, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I would add just one spec that tests that you can intantiate a model and expect that it responds to methods 

see: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/respond-to-matcher